spec/haskell/src/SEL4/Kernel/Thread.lhs

```diff
@@ -526,28 +526,14 @@ The following function is used to alter the priority of a thread.
 >     case ts of
 >          Running -> threadSetPriority_onRunning tptr prio
 >          Restart -> threadSetPriority_onRunning tptr prio
->          BlockedOnSend ep _ _ _ _ -> threadSetPriority_onEp tptr prio ep
```

Comment:
Has anyone mention that this change might make `threadSetPriority_onEp` unused? Has it been removed?